semgrep/Pipfile

```diff
@@ -12,6 +12,7 @@ appdirs = "*"
 pytest-xdist = "*"
 
 [packages]
+jsonschema = "~=3.2.0"
```

Comment:
I believe we include dependencies in `setup.py` then rely on `path = "."` below for local installation.